src/gui/accountsettings.cpp

```diff
@@ -55,13 +55,30 @@
 #include <QVariant>
 #include <QJsonDocument>
 #include <QToolTip>
-#include <qstringlistmodel.h>
-#include <qpropertyanimation.h>
 
 #include "account.h"
 
 namespace {
 constexpr auto propertyFolderInfo = "folderInfo";
+
+bool showEnableE2eeWithVirtualFilesWarningDialog(QObject *parent)
+{
+    Q_ASSERT(parent);
+    QMessageBox e2eeWithVirtualFilesWarningMsgBox;
+    e2eeWithVirtualFilesWarningMsgBox.setText(parent->tr("End-to-End Encryption with Virtual Files"));
```

Comment:
Better either use `AccountSettings::tr()` (this is static) or `QCoreApplication::translate()`. Please don't pass an (in fact) unused instance.

https://doc.qt.io/qt-5/i18n-source-translation.html#using-tr-for-all-literal-text

Reply:
Good catch indeed.

Reply:
Thanks. I used `QObject::tr()` now. That should work also I think? Wanted to make it more general and not dependent on `AccountSettings`. `QCoreApplication::translate()` is more complicated to call.